cmd/cluster/azure/create.go

```diff
@@ -90,6 +93,31 @@ type ValidatedCreateOptions struct {
 }
 
 func (o *RawCreateOptions) Validate(ctx context.Context, opts *core.CreateOptions) (core.PlatformCompleter, error) {
+	var err error
+
+	// Validate values used for PayloadArch
+	if o.PayloadArch != "Single" && o.PayloadArch != "Multi" {
+		return nil, fmt.Errorf("unsupported payload architecture type %s", o.PayloadArch)
+	}
+
+	// Perform multi-arch validations except for render
+	isMultiPayload := o.PayloadArch == "Multi"
+	if !opts.Render {
+		isMultiPayload, err = core.ValidateMultiArch(ctx, isMultiPayload, opts)
```

Comment:
why do we have ValidateMultiArch and ValidateMultiArchRelease? what's the difference?

Reply:
One was a newer version written during the course of this PR. It's been removed and there is only one method now.